### PR TITLE
Normalize usage of directory vs folder.

### PR DESCRIPTION
**Pull request by MC42**
In reference to #1065 .

### CODE REVIEW

File: application/pages/InstanceSettingsPage.ui [hunk 1]
```diff
@@ -11,16 +11,7 @@
...
-    <number>0</number>
-   </property>
-   <property name="topMargin">
-    <number>0</number>
-   </property>
-   <property name="rightMargin">
-    <number>0</number>
-   </property>
-   <property name="bottomMargin">
+   <property name="margin">
```
Comment (peterix) [inline application/pages/InstanceSettingsPage.ui]: This change seems to be unnecessary. I'm also not sure if Qt 5.4 tools can handle it (I haven't tried).

File: application/pages/global/JavaPage.ui [hunk 2]
```diff
@@ -17,16 +17,7 @@
...
-    <number>0</number>
-   </property>
-   <property name="topMargin">
-    <number>0</number>
-   </property>
-   <property name="rightMargin">
-    <number>0</number>
-   </property>
-   <property name="bottomMargin">
+   <property name="margin">
```
Comment (peterix) [inline application/pages/global/JavaPage.ui]: Also unnecessary.

File: application/pages/InstanceSettingsPage.ui (hunk 1 above)
Comment (MC42) [inline application/pages/InstanceSettingsPage.ui]: Reverted and simply edited in vim as per request.

File: application/pages/InstanceSettingsPage.ui [hunk 3]
```diff
@@ -29,7 +20,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
```
Comment (MC42) [inline application/pages/InstanceSettingsPage.ui]: Reverted and simply edited in vim as per request.

File: application/pages/global/JavaPage.ui (hunk 2 above)
Comment (MC42) [inline application/pages/global/JavaPage.ui]: Reverted and simply edited in vim as per request.

**Comment by peterix** [comment]
I expected you would actually squash/fix the commit messages. I guess I should explain better next time :)

Anyway, rebased, squashed, reworded the commit message and merged.

**Pull request merged**